python/tests/integration/completions/test_text_completion.py

```diff
@@ -19,26 +25,19 @@
     OpenAITextCompletion,
     OpenAITextPromptExecutionSettings,
 )
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
 from semantic_kernel.connectors.ai.text_completion_client_base import TextCompletionClientBase
 from semantic_kernel.contents.chat_message_content import ChatMessageContent
 from semantic_kernel.contents.text_content import TextContent
 from semantic_kernel.utils.authentication.entra_id_authentication import get_entra_auth_token
-
-if sys.version_info >= (3, 12):
-    from typing import override  # pragma: no cover
-else:
-    from typing_extensions import override  # pragma: no cover
-
-from semantic_kernel import Kernel
-from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
 from tests.integration.completions.completion_test_base import CompletionTestBase, ServiceType
 from tests.utils import is_service_setup_for_testing, is_test_running_on_supported_platforms, retry
 
-ollama_setup: bool = is_service_setup_for_testing(["OLLAMA_TEXT_MODEL_ID"]) and is_test_running_on_supported_platforms([
-    "Linux"
-])
-google_ai_setup: bool = is_service_setup_for_testing(["GOOGLE_AI_API_KEY"])
-vertex_ai_setup: bool = is_service_setup_for_testing(["VERTEX_AI_PROJECT_ID"])
+ollama_setup: bool = is_service_setup_for_testing(
+    ["OLLAMA_TEXT_MODEL_ID"], raise_if_not_set=False
```

Comment:
If we do not raise an error when the service is not set up in our workflow, we may never know if a particular service is being tested or not.